src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -208,6 +220,49 @@ private BidRequest createSingleRequest(Imp imp, BidRequest bidRequest, boolean u
                 .build();
     }
 
+    private String makeUri(BidRequest bidRequest) {
+        final String tkXint = tkXintValue(bidRequest);
+        if (!StringUtils.isBlank(tkXint)) {
```

Comment:
There is companion method in `StringUtils` called `isNotBlank` - using it will improve readability.